pwiz_tools/Skyline/CommandLine.cs

```diff
@@ -813,6 +813,12 @@ private bool PerformExportOperations(CommandArgs commandArgs)
                 return false;
             }
 
+            if (commandArgs.ExportingMProphetFeatures && 

```

Comment:
In this case, I prefer the nested ifs to make it more readable. Also, avoid KR style bracing. We use matching braces.